tools/mason/MasonPublish.chpl

```diff
@@ -690,7 +690,9 @@ proc refreshLicenseList(overwrite=false) throws {
   const branch = '--branch main ';
   const depth = '--depth 1 ';
   const url = 'https://github.com/spdx/license-list-data.git ';
-  const command = 'git clone -q ' + branch + depth + url + dest;
+  const referIfAble = if MASON_LICENSE_CACHE_PATH != "" then
```

Comment:
The approach we've used in other tests so far is to always pass `--reference-if-able`, and just use a sentinel value `/missing` if the env var isn't set so it's easy to find in logs. I think that would be good to do here as well

Reply:
Won't that lead to paratest failures for team members unless we all start setting the environment variable?

Reply:
No, `--reference-if-able` only emits a warning if the path it's given isn't a valid git repo. There's a corresponding version `--reference` which would error.

Reply:
Actually, yeah maybe just the warning getting emitted would break things since it's extra output we compare to `.good` (and I don't know if `-q` silences this warning or not). Maybe best to leave as you have it

Reply:
It also would probably cause problems for users unless they set the environment variable and I don't think we want to require that of them.  Thanks!